src/mysql/populate.sql

```diff
@@ -0,0 +1,100 @@
+-- This file will populate the database with dummy data
+
+USE Mintern;
+
+INSERT INTO Major (name) 
+VALUES 
+  ('ITC'),
+  ('INT'),
+  ('ISD'),
+  ('ITI'),
+  ('IMT');
+
+INSERT INTO User (name, email, major_id, is_mentor) 
+VALUES 
+  ('Shaar', 'a00825287@itesm.mx', 1, TRUE),
+  ('Andres', 'a01283152@itesm.mx', 3, TRUE),
+  ('Omar', 'a01206177@itesm.mx', 1, TRUE),
+  ('Ernesto', 'a00825923@itesm.mx', 1, FALSE),
+  ('Diego', 'a00824758@itesm.mx', 1, FALSE),
+  ('Rodrigo', 'a01234245@itesm.mx', 1, FALSE),
+  ('Daniel', 'a00825596@itesm.mx', 1, FALSE);
+
+INSERT INTO Notification (user_id, message, url, date_time) 
+VALUES
+  (4, 'You got an answer', 'questions/1', '2020-07-29 13:00:00.000000'),
+  (6, 'You got an answer', 'questions/2', '2020-07-29 14:00:00.000000'),
+  (6, 'You got an answer', 'questions/2', '2020-07-29 15:00:00.000000'),
+  (5, 'You got an answer', 'questions/3', '2020-07-29 16:00:00.000000'),
+  (7, 'You got an answer', 'questions/4', '2020-07-29 17:00:00.000000'),
+  (7, 'You got an answer', 'questions/4', '2020-07-29 18:00:00.000000'),
+  (3, 'Somebody commented your answer', 'questions/2', '2020-07-29 19:00:00.000000'),
+  (3, 'Somebody commented your answer', 'questions/2', '2020-07-29 20:00:00.000000');
+
+INSERT INTO Tag (title, color) 
```

Comment:
It may make your life easier if you use HEX colors instead of typing out the color name. If you type out the color name, then you have to worry more about misspellings, capitalization, etc. Something to consider!

Reply:
done